module/zfs/vdev_raidz.c

```diff
@@ -1440,6 +1440,38 @@ vdev_raidz_child_done(zio_t *zio)
 	rc->rc_skipped = 0;
 }
 
+boolean_t
+vdev_raidz_need_resilver(vdev_t *vd, uint64_t offset, size_t psize)
```

Comment:
I would suggest adding a comment explaining what this function is supposed to do, and its assumptions.  For example, it assumes that the at least one of the children has a dirty dtl.